### PR TITLE
on-tasks code

### PR DESCRIPTION
**Pull request by longhornteam35**
on-tasks code additions. Includes tasks and jobs for create volume,
delete volume, and add hotspare.

### CODE REVIEW

File: lib/jobs/add-volume.js [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+                //logger.info(results);
+                logger.info("create vd job is done");
+                self._done();
+            })
+            .catch(function(err) {
+                self._done(err);
+            }); 
+
+     	self._done();
+     }
```
Comment (rahmanmuhamad) [inline lib/jobs/add-volume.js]: Please have these houndCI comments fixed . some are important

File: lib/jobs/delete-volume.js [hunk 2]
```diff
@@ -0,0 +1,109 @@
+'use strict'
```
Comment (rahmanmuhamad) [inline lib/jobs/delete-volume.js]: we need to fix this.

**Comment by rahmanmuhamad** [comment]
Plz have these comments and suggestion fixed,

**Comment by iceiilin** [comment]
As the three jobs looks quite similar, is there possible to use only one job and distinguish their logic in JobUtils.RacadmTool? Or maybe we could leverage an existing racadm contorl base task (https://github.com/RackHD/on-tasks/blob/master/lib/task-data/base-tasks/racadm-control-base.js), and only implement corresponding tasks, just like https://github.com/RackHD/on-tasks/blob/master/lib/task-data/tasks/dell-racadm-update-firmware.js? Their actual logic can be added here, just like https://github.com/RackHD/on-tasks/blob/master/lib/utils/job-utils/racadm-tool.js#L222

**Comment by dalebremner** [comment]
It would be good to have unit tests

**Comment by rahmanmuhamad** [comment]
Don't have access to the repo to make these changes, i will do it after merge

**Comment by michaelsteven** [comment]
Agree with the plan.  We do not have write access to the Longhorn 35 organization and the semester is over.  It is unknown if or when we would gain access.

**Pull request merged**